cookie-manager.js

```diff
@@ -0,0 +1,228 @@
+"use strict";
+
+(function () {
+
+    const init = function () {
+        readConfiguration();
+    };
+
+    const readConfiguration = function() {
+
+        const xhr = new XMLHttpRequest();
+        xhr.open('GET', '/cookie-manager.json');
```

Comment:
any value in having this manifest location as a property - which could be overridden when calling the code

Reply:
Currently, the script is a self invoking function. So once the script is imported, its already initialising; this approach was opted for as it would be the easiest approach to use the script, as users of the library would just have to have a `cookie-manager.json` in the root of the site (in the same place as a `robots.txt`. And just include the script into their markup with no invocation required. But it would be good to have a discussion about this.

Reply:
Though, we could definitely extract the path to a variable as we use the path in messages elsewhere.

Reply:
Turns out, that after discussion. We remove the requirement for a separate file altogether.

Instead, we initialise the class with the configuration, which overrides some defaults.